### PR TITLE
CIV-10058 hwf caseworker no remission hwf

### PR DESCRIPTION
**Pull request by jeswanth-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-10058

### Change description ###

 hwf caseworker no remission hwf

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/CaseEventToFields/NoRemission-HWF-CUI-R2-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "DisplayContext": "READONLY",
+    "PageID": "NoRemissionHWF",
+    "PageDisplayOrder": 1,
+    "PageFieldDisplayOrder": 2,
+    "FieldShowCondition": "caseTitleWithHWFFeeNumber=\"DO_NOT_SHOW_IN_UI\""
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "NO_REMISSION_HWF",
+    "CaseFieldID": "caseTitleWithHWFFeeNumber",
```
Comment (manish14836) [inline ccd-definition/CaseEventToFields/NoRemission-HWF-CUI-R2-nonprod.json:25]: Change this to caseTitleWithHWFClaimFeeNumber

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-HWF-R2-CUI-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "NO_REMISSION_HWF",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-admin",
+          "cui-admin-profile"
```
Comment (manish14836) [inline ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-HWF-R2-CUI-nonprod.json:9]: I think you need to give access to  "caseworker-civil" also. plz check
Does "cui-admin-profile" need access, as per story it should be case worker only? Applicable to all places where "cui-admin-profile" or any other role except  "caseworker-civil-admin", "caseworker-civil", is used.

File: ccd-definition/CaseEventToFields/NoRemission-HWF-CUI-R2-nonprod.json (hunk 1 above)
Comment (jeswanth-hmcts) [reply ccd-definition/CaseEventToFields/NoRemission-HWF-CUI-R2-nonprod.json]: changed the name has mentioned

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-HWF-R2-CUI-nonprod.json (hunk 2 above)
Comment (jeswanth-hmcts) [reply ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-HWF-R2-CUI-nonprod.json:8]: added only cui-admin-profile has mentioned

**Review by portizdev — APPROVED** [review]

File: ccd-definition/ComplexTypes/HelpWithFeesDetails-CUI-R2-nonprod.json.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "ID": "HelpWithFeesDetails",
+    "ListElementCode": "noRemissionDetails",
+    "ElementLabel": "Details",
+    "FieldType": "Text",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
```
Comment (ElifDenizEsea) [inline ccd-definition/ComplexTypes/HelpWithFeesDetails-CUI-R2-nonprod.json.json:8]: helpwithfeetype used in conditions but missing here

**Review by sankhajuria — APPROVED** [review]

**Review by kalachandrasekar1 — APPROVED** [review]

**Review by portizdev — APPROVED** [review]

**Pull request merged**